modules/richards/tests/sinks/tests

```diff
@@ -5,6 +5,14 @@
     csvdiff = 's01.csv'
     rel_err = 1.0E-5
   [../]
+  [./s01_permch]
+    type = 'CSVDiff'
+    input = 's01.i'
+    cli_args = 'Materials/rock/perm_change="one one one one one one one one one" Materials/rock/mat_permeability="1E-6 0 0 0 1E-6 0 0 0 1E-6"'
```

Comment:
hahahah.  now i'm trying to think where i can use some sort of "cody" parameter.  Thanks for the merge,

a